src/main/webapp/scripts/ua_kuali_application.js

```diff
@@ -57,9 +57,7 @@ function loadLeadUnitName(unitNumberFieldName, leadUnitNameFieldName ) {
 	var unitNumber = dwr.util.getValue( unitNumberFieldName );
 	clearRecipients( leadUnitNameFieldName, "" );
 
-	if (unitNumber=='') {
```

Comment:
Changes look good, there's still no additional logging for the error if the call to the unit service fails,however setting the window.status to the error message might suffice for debugging purposes.